cron/poll_issue_close_stale.py

```diff
@@ -0,0 +1,45 @@
+import arrow
+import logging
+
+import settings
+import github_api as gh
+
+__log = logging.getLogger("poll_close_stale")
+
+
+def poll_issue_close_stale():
+    """
+    Looks through all open issues. For any open issue, if the issue is
+    too old and has not been recently commented on, chaosbot issues a
+    /vote close...
+    """
+
+    __log.info("Checking for stale issues...")
+
+    api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
```

Comment:
one thing I noticed that may be a potential source of api rate limit bugs.  When a new api instance is created, it sets its remaining requests to infinity, and so assumes that it can proceed with any request and will succeed.  Sometimes this may not succeed if we're low on api requests though.  One solution I can think of is use a single api instance at the top-level cron module and pass it into each cron polling function, or (if it's difficult to pass arguments to be called with the callback) bake it into the function with [functools.partial](https://docs.python.org/3.6/library/functools.html#functools.partial)